packages/tatum-algo/src/wallet/wallet.ts

```diff
@@ -19,11 +19,9 @@ export const generateAlgoWallet = async (mnem?: string): Promise<WalletWithAddre
 
 /**
  * Generate wallet
- * @param currency blockchain to generate wallet for
- * @param testnet testnet or mainnet version of address
- * @param mnemonic mnemonic seed to use. If not present, new one will be generated
+ * @param options.mnemonic mnemonic seed to use. If not present, new one will be generated
  * @returns wallet or a combination of address and private key
  */
-export const generateWallet = (mnemonic?: string) => {
-  return generateAlgoWallet(mnemonic)
+export const generateWallet = (options: { mnemonic?: string }) => {
```

Comment:
Does it also work without arguments? I mean just `generateWallet()` instead of `generateWallet({})` (empty options object)

Reply:
Right here in this case it will work as there is used optional chaining, turning options to undefined if no argument passed.